### DISCUSSION TITLE
Revert "Place Order" button back to enabled

### DISCUSSION THREAD

**Discussion by shlomtzion1980** [woocommerce/woocommerce#49213] [Checkout Flow]
Currently , 
We are creating either a popup/inpage/next-page iframe or redirect, through the new blocks.
If you decide to close the iframe and perhaps to choose a different payment method we are currently reloading which is not the wanted action.

How can we revert the "Place Order" button to become active again if our iframe is closed?

**Comment by nielslange** [comment]
Hello @shlomtzion1980 👋

To answer your question, can you describe your case in more detailed? What exactly are you trying to accomplish?

**Reply by jimmymnt** [reply]
Hi, 
I have same problem above.

For example, in the Checkout page (For Block), when pressing "Place Order" button, our process will handle some logic, open a Modal and fill some requirement input, etc..., and after that the order will be created but pending the the system to redirect to THANKYOU page, or perhaps - our customer make some miss typo and close , at that moment the "Place Order" button will be disabled and we want to revert it back to ENABLE to change the mistake.


<img width="832" alt="image" src="https://github.com/user-attachments/assets/920ef60e-bf02-46bc-9173-50608aef919a">

**Comment by shlomtzion1980** [comment]
Say we load an iframe popup in the page for cc payment, and the customer
decided to close the iframe and choose a different payment method - a
different iframe or something like that...

Currently in order to allow him to choose it i need to reload the page...

Can't i reset the code?button?

In classic checkout the same button opens the popup but if you choose to
close it and choose another method it will allow you to do so without
reload.
